apisix/admin/resource.lua

```diff
@@ -49,7 +49,41 @@ local function split_typ_and_id(id, sub_path)
 end
 
 
-function _M:check_conf(id, conf, need_id, typ)
+local function check_create_update_time(conf)
```

Comment:
Why do you need to differentiate each resource?
Why not do like the function `utils.iinject_timestamp`?

Reply:
Logically speaking, it is not allowed to pass in create_update and update_time when creating any core resources. I just expanded the check scope of the check_conf function to comply with the original intention of this function design (I understand)

Reason for not modifying `inject_timestamp`
1. This function does not check the subsource design. For example, when creating a route resource, the route resource contains the upstream resource. At this time, the upstream resource has create_time.